conf/openmetadata.yaml

```diff
@@ -241,6 +241,7 @@ elasticsearch:
   scheme: ${ELASTICSEARCH_SCHEME:-http}
   username: ${ELASTICSEARCH_USER:-""}
   password: ${ELASTICSEARCH_PASSWORD:-""}
+  clusterAlias: ${ELASTICSEARCH_CLUSTER_ALIAS:-""}
```

Comment:
Can we use a more concrete name for the config and environment variable like `index` and `ELASTICSEARCH_INDEX`?

Reply:
ELASTICSEARCH_INDEX means something else for users.